docs/api/context-bridge.md

```diff
@@ -110,3 +110,16 @@ has been included below for completeness:
 | `Symbol` | N/A | ❌ | ❌ | Symbols cannot be copied across contexts so they are dropped |
 
 If the type you care about is not in the above table, it is probably not supported.
+
+### Exposing Node Global Symbols
+
+The `contextBridge` can be used by the preload script to give your renderer access to Node global symbols.
+The table of supported types described above also applies to Node globals that you expose through `contextBridge`.
+Please note that many Node APIs grant access to local system resources.
+Be very cautious about which globals and APIs you expose to untrusted remote content.
+
+```javascript
+const { contextBridge } = require('electron')
+contextBridge.exposeInMainWorld('setImmediate', setImmediate)
+contextBridge.exposeInMainWorld('clearImmediate', clearImmediate)
```

Comment:
Perhaps we could have a more useful/relevant example than `setImmediate` here? And perhaps also show an example of not directly exposing Node stuff, but rather exposing a safe wrapper.

```suggestion
const crypto = require('crypto')
contextBridge.exposeInMainWorld('nodeCrypto', {
  sha256sum(data) {
    const hash = crypto.createHash('sha256')
    hash.update(data)
    return hash.digest('hex')
  }
})
```